CHANGELOG.md

```diff
@@ -79,6 +79,7 @@
         * Remove `ThreeDSecureV2BaseCustomization`
         * Remove `CardinalValidateReceiver` from `ThreeDSecureActivity`
         * Make empty `ThreeDSecureLookup` constructor package-private
+        * Add `setCustomFields` to `ThreeDSecureRequest`
```

Comment:
```suggestion
```

I think we can just copy over the entire `4.48.0` CHANGELOG (already done in another PR) to include this change since it's not new in v5